### DISCUSSION TITLE
Support for anchors

### DISCUSSION THREAD

**Discussion by pr1m0x** [vercel/next.js#23746] [Help]
Hey,

are page anchors supported in NextJS? (jumping to an specific element/container based on #id)

`
<a href="#id">
<element id="id">
`

Cheers

**Comment by akellbl4** [comment]
Next.js just a framework on top of React. It means you can do the same things that you've done with React. Even more, you can do everything is provided by the browser. [Here is an example working example.](https://codesandbox.io/s/next-js-anchors-p99vr?file=/pages/index.js)
